tasklib/backends.py

```diff
@@ -427,3 +427,34 @@ def valid(output):
 
     def sync(self):
         self.execute_command(['sync'])
+
+    def _get_task_attrs(self):
+        TASK_STANDARD_ATTRS = [
+            'annotations',
+            'entry',
+            'depends',
+            'description',
+            'due',
+            'end',
+            'imask',
+            'mask',
+            'modified',
+            'parent',
+            'priority',
+            'project',
+            'recur',
+            'scheduled',
+            'start',
+            'status',
+            'tags',
+            'until',
+            'uuid',
+            'wait',
+        ]
+        available_task_attrs = TASK_STANDARD_ATTRS
+        udas = set()
+        for index in self.config:
+            if 'uda' in index:
+                udas.add(re.sub(r'.*uda\.(.*?)\..*', r'\1', index))
+        available_task_attrs.extend(udas)
+        self.available_task_attrs = tuple(available_task_attrs)
```

Comment:
Since the method is a "getter", it should return something rather than just set an attribute on the object.

Reply:
You are right, I changed it to `_set_task_attrs` on 7805380